.travis/smoke-test.sh

```diff
@@ -9,5 +9,9 @@ trap finish EXIT
 
 make start wait-healthy
 
-ping=$(curl -sS 'http://localhost:8080/' 2>&1)
-[[ $ping == 'OK' ]]
+output=$(curl --silent --show-error --write-out '%{http_code}' 'http://localhost:8080/')
+body=${output::${#output}-3}
```

Comment:
very obscure; `echo "$output" | head -n 1` and `echo "$output" | tail -n 1`?

Reply:
Can't assume the body is one line.

Reply:
Ah, meant `head -n -1` (print all but the last 1 line[s])